include/feeding/action/FeedFoodToPerson.hpp

```diff
@@ -12,29 +12,10 @@ namespace feeding {
 namespace action {
 
 void feedFoodToPerson(
-    const std::shared_ptr<ada::Ada>& ada,
-    const std::shared_ptr<Workspace>& workspace,
-    const aikido::constraint::dart::CollisionFreePtr& collisionFree,
-    const aikido::constraint::dart::CollisionFreePtr&
-        collisionFreeWithWallFurtherBack,
     const std::shared_ptr<Perception>& perception,
     const ros::NodeHandle* nodeHandle,
     const Eigen::Isometry3d& plate,
```

Comment:
Note for later that I'm not sure why we would need the plate transform here, except that is is necessary for the final MoveAbovePlate action (if that's in here). If it is, we should separate it out.

See below comment as well, but I think we should have separate "FeedFoodToPerson", "Tilt Food", and "Return To Plate" actions, which should make this easier to understand.

Reply:
Yup, this behemoth of an action does the whole bite transfer and then returns to the plate. Agreed that "Return to Plate" should be separated out. 

I'm not sure whether "Tilt Food" should be separated from "FeedFoodToPerson" because imo it is all part of the bite transfer and the reason to tilt the food is to facilitate feeding it to the person. Further, I could envision bite transfer trajectories where the timing of tilting matters, and therefore you can't tilt the food independently from the bite transfer.

Regardless, we can/should discuss this later for when we actually split off the actions.